### DISCUSSION TITLE
Simple example

### DISCUSSION THREAD

**Discussion by victor999** [LeoNatan/LNPopupController#516] [General] [answered]
It will be great to see some simple example for swift app. Like app consists of 2 View Controllers and Tab bar controller. I read the docs, looked at the examples provided and still struggle to implement it.

**Comment by LeoNatan** [comment, marked as answer]
Hi Victor,
Do you mind sharing what you have attempted and failed?
The docs show you exactly what you need to do, in order to display a view controller as the popup content controller from a tab bar controller:
```swift
//Load the content controller by initializing it, from a storyboard, or any other way
let demoVC = DemoPopupContentViewController()

//Set the popup item, so that content appears on the popup bar:
demoVC.popupItem.title = "Hello World"
demoVC.popupItem.subtitle = "And a subtitle!"
demoVC.popupItem.progress = 0.34

//Present the popup bar from the tab bar controller, with content from the demoVC content view controller:
tabBarController?.presentPopupBar(withContentViewController: demoVC, animated: true, completion: nil)
```

**Reply by victor999** [reply]
I managed to get it to work, thank you for your explanation
I created a repository with this example, I hope it's OK
https://github.com/victor999/TabBar-App
